src/controllers/articles.js

```diff
@@ -0,0 +1,96 @@
+import dotenv from 'dotenv';
+import { Article } from '../database/models';
+
+dotenv.config();
+
+export default {
+  create(req, res) {
+    const { title, description, content, image, slug } = req.body;
+    return Article.create({
+      ...req.body
+    })
+      .then(article =>
+        res.status(201).send({
+          status: 201,
+          data: article
+        })
+      )
+      .catch(error => res.status(400).send(error));
+  },
+
+  list(req, res) {
+    return Article.findAll()
+      .then(articles =>
+        res.status(200).send({
+          status: 200,
+          data: articles
+        })
+      )
+      .catch(error => res.status(400).send(error));
+  },
+
+  retrieve(req, res) {
+    return Article.findByPk(req.params.articleId, {})
+      .then(article => {
+        if (!article) {
+          return res.status(404).send({
+            message: 'Article Not Found'
+          });
+        }
+        return res.status(200).send({
+          status: 200,
+          data: article
+        });
+      })
+      .catch(error => res.status(400).send(error));
+  },
+
+  update(req, res) {
+    return Article.findByPk(req.params.articleId, {})
+      .then(article => {
+        if (!article) {
+          return res.status(404).send({
+            message: 'Article Not Found'
+          });
+        }
+        return article
+          .update({
+            title: req.body.title || article.title,
```

Comment:
Nice implementation but I think you could destructure the `article` and the `req.body` first to make your code cleaner